tests/test_views.py

```diff
@@ -333,17 +332,12 @@ def test_post_patch_real_social_auth_enforced(
         mock_get_ec.return_value = customer
         mock_get_ec2.return_value = customer
         mock_get_rsa.return_value = mock.MagicMock(user=UserFactory())
-        with raises(NoReverseMatch) as excinfo:
+        with self.assertRaises(NoReverseMatch):
             client = Client()
             session = client.session
             session['partial_pipeline_token'] = True
             session.save()
             client.post(self.url)
-        expected = (
-            'Reverse for \'dashboard\' with arguments \'()\' and keyword '
-            'arguments \'{}\' not found. 0 pattern(s) tried: []'
-        )
-        assert str(excinfo.value) == expected
```

Comment:
This error message is no longer guaranteed -- different types of messages may be output. In the case where arguments and keywords are empty (as this was expecting), this is no longer the expected error message. Decided to just scratch hardcoding them throughout the codebase and look more importantly for the assertion.